app/lib/server/server.go

```diff
@@ -319,10 +324,10 @@ func entityListToPublics(el *sda.Roster) []abstract.Point {
 	return publics
 }
 
-func isPublicIP(ip string) bool {
+func isPublicIP(ip network.Address) bool {
```

Comment:
Seems to be implemented here: https://github.com/dedis/cothority/pull/567/commits/9125dfdb38993916d664ae38e305a89e5c48bb7a